proxy-robots-module/src/main/java/orwell/proxy/controller/GameGUI.java

```diff
@@ -0,0 +1,98 @@
+package orwell.proxy.controller;
+
+import lejos.mf.common.UnitMessage;
+import lejos.mf.common.UnitMessageType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.robot.IRobot;
+import orwell.proxy.robot.MessageNotSentException;
+
+import javax.swing.*;
+import java.awt.event.*;
```

Comment:
I have tried by importing them all one by one, but with swing and awt it is really painful, and probably not worth the bother since this is just for occasional testing purpose. Besides, even my IDE wants to optimize it this way - I do not dare to oppose it :)